tests/basic/pool_function_test.go

```diff
@@ -833,3 +834,87 @@ var _ = Describe("{PoolExpandAddDiskInMaintenanceMode}", func() {
 		})
 	})
 })
+
+var _ = Describe("{PoolVolUpdateResizeDisk}", func() {
+
+	var testrailID = 34542876
+	// testrailID corresponds to: https://portworx.testrail.net/index.php?/tests/view/34542876
+
+	BeforeEach(func() {
+		StartTorpedoTest("PoolVolUpdateResizeDisk", "expand volume to the pool and pool expansion using resize-disk", nil, testrailID)
+		contexts = scheduleApps()
+	})
+	JustBeforeEach(func() {
+		poolIDToResize = pickPoolToResize()
+		log.Infof("Picked pool %s to resize", poolIDToResize)
+		storageNode, err = GetNodeWithGivenPoolID(poolIDToResize)
+		log.FailOnError(err, "Failed to get node with given pool ID")
+
+	})
+	JustAfterEach(func() {
+		AfterEachTest(contexts)
+	})
+
+	AfterEach(func() {
+		appsValidateAndDestroy(contexts)
+		EndTorpedoTest()
+	})
+	stepLog := "Expand volume to the pool and pool expansion using resize-disk"
```

Comment:
An `It` block declares a test case, and its first string parameter is the test description, not a "stepLog".
The current usage is not technically "incorrect" for your `stepLog` does explain the purpose of this test (though the wording needs improvement for clarify), but let's aim to use the framework properly.    
Please spend a few minutes reading the documentation https://onsi.github.io/ginkgo/